src/SFA.DAS.ApprenticeFeedback.Domain/Models/Feedback/FeedbackAttribute.cs

```diff
@@ -1,13 +1,16 @@
-﻿namespace SFA.DAS.ApprenticeFeedback.Domain.Models
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace SFA.DAS.ApprenticeFeedback.Domain.Models.Feedback
 {
     public class FeedbackAttribute
     {
         public int Id { get; set; }
         public string Name { get; set; }
         public string Category { get; set; }
         public int Ordering { get; set; }
+
+        [Required(ErrorMessage = "Select Agree or Disagree")]
```

Comment:
On line 41 of FeedbackAttributes.cshtml.cs, there's logic that looks like it might depend on the exact wording of the string "Select Agree or Disagree".
`ModelState.AddModelError("MultipleErrorSummary", "Select Agree or Disagree");`

We can avoid this coupling by storing this error message as a `const string` somewhere instead of hardcoding it in these places.

Reply:
Have created a shared const

Reply:
Resolving this in the interests of time